docs/advanced/transaction-details.md

```diff
@@ -2,11 +2,28 @@
 
 ---
 
-Decred transactions are transfers of DCR that exist within blocks. Transactions are comprised primarily of inputs and outputs, though they have a few other fields of data as well. 
+Decred transactions are transfers of DCR that exist within blocks. Transactions are comprised primarily of inputs and outputs, though they have a few other fields of data as well. In this page, we will examine the types of transactions Decred supports, the basic format common to all transactions, and examine an example transaction in detail. 
+
+
+## Transaction Types
+
+To enable Decred's consensus rules, transactions are split into two separate trees within a block: the regular transaction tree and the stake transaction tree. The stake transaction tree contains transactions related to ticket buying and ticket voting rewards. The regular transaction tree contains all other transactions. 
+
+In addition to regular transactions (sending DCR from one address to another), there are a few special transaction types to be aware of:
+
+* **Coinbase transaction:** Every block which is mined contains a single coinbase transaction. This transaction will only have one input, and that is newly created Decred which constitutes the PoW and Treasury portions of the block reward. Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if Proof-of-Stake (PoS) voters vote to reject the block which contains them. Decred created in coinbase transactions cannot be spent until the coinbase maturity period has passed.
+* **Stakebase transaction:** Every block which is mined will contain a stakebase transaction for each ticket which voted on that block. Stakebase transactions are located in the stake transaction tree and have two inputs: the Decred which was spent to purchase the ticket, and the newly created Decred constituting the reward for voting. Stakebase transactions are also created when an expired or missed ticket is revoked, however there will be no voting reward created in these cases. Stakebase transactions belong to the stake transaction tree, which means that these transactions cannot be rejected by PoS voters, even if they vote to reject the block which contains them.
```

Comment:
Every block starting at the stake validation height which is mined ...

Reply:
Same here.  The inputs don't create the coins, rather it's the outputs.

Reply:
s/reject/disapprove/

Reply:
The GitHub UI is lumping all three comments together, making it difficult to see which line each comment is pointing to, so just going to paste the edits I made below, in case anything jumps out as still wrong. Bold is to highlight changes, not in actual page. 

>Stakebase transaction: Every block **starting at the stake validation height** which is mined

>This transaction will only have one **output**, and that is newly created Decred which constitutes the PoW.. 

This one is made in two places:

>Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if Proof-of-Stake (PoS) voters vote to **reject/disapprove** the block which contains them.

>Stakebase transactions belong to the stake transaction tree, which means that these transactions cannot be rejected by PoS voters, even if they vote to **reject/disapprove** the block which contains them.

Reply:
That was programmer speak for replace "reject" with "disapprove".  You can't vote to reject a block, only to disapprove it.  Actually rejecting a block is accomplished by not casting a vote at all.

Reply:
Ah. Added to my developer to English dictionary. 

>Actually rejecting a block is accomplished by not casting a vote at all.

Good to know. There was some confusion about this terminology on Matrix the other day.